### DISCUSSION TITLE
本地文件图片预览如何打开？

### DISCUSSION THREAD

**Discussion by Misaka-Mikoto-Tech** [AlistGo/alist#1861] [Q&A]
我用docker安装了v3.0.3, 挂载了本地目录，并在存储界面中打开了缩略图的开关，一开始图片可以显示缩略图，但我添加了若干个存储并设置了其它选项后发现缩略图都没了，并且画廊模式下方的图片列表都加载不出来了，请问有知道应该如何设置才能让本地目录显示缩略图预览的吗？

**Comment by Misaka-Mikoto-Tech** [comment]
用户打开带图片的目录时，后台临时生成缩略图也行，就像资源管理器第一次打开带图片的文件夹时的效果（一张张的图刷出来）

**Comment by monsterxcn** [comment]
我好像也是，我是用 docker [xhofe/alist:latest](https://registry.hub.docker.com/layers/xhofe/alist/latest/images/sha256-1dc600f99b0959082c9cb9851e176210a0e59d4fdd6c06252805ce2b8aec5231) 搭的，映射了一个本地目录进去

我记得刚搭好那会挂载了本地目录，还能看见本地图片的缩略图来着。整完其他的存储之后发现只有阿里云盘、OneDrive 能显示缩略图了，本地、OSS、COS 都没有缩略图。

我去逛了逛别的 discussion，得到的答案是：

> 所有的缩略图都是由对应的存储提供的，只有对应的存储提供了缩略图才会显示缩略图的。
> 
> Originally posted by @Xhofe in https://github.com/alist-org/alist/discussions/372#discussioncomment-1976717_


> 是网盘的问题不是本程序控制的
> 
> Originally posted by @anwen-anyi in https://github.com/alist-org/alist/discussions/1058#discussioncomment-3064832_

就感觉很怪，本地目录有什么特别之处吗？刚开始好像真的有那么一下缩略图...

**Reply by monsterxcn** [reply]
嗷嗷，我看见了，本机存储直接设置一下开关就好了：

![image](https://user-images.githubusercontent.com/22407052/197154955-afc19f8f-8fd3-4cc7-9aa7-e6f49437d6f4.png)

COS、OSS 要是也有这个就好了哈哈

**Reply by Misaka-Mikoto-Tech** [reply]
> 嗷嗷，我看见了，本机存储直接设置一下开关就好了：
> 
> ![image](https://user-images.githubusercontent.com/22407052/197154955-afc19f8f-8fd3-4cc7-9aa7-e6f49437d6f4.png)
> 
> COS、OSS 要是也有这个就好了哈哈

我打开了这个选项也不管用

**Reply by mohui1999** [reply]
我也是，打开开关也不能生成缩略图

**Reply by DreamlingBig** [reply]
> 我也是，打开开关也不能生成缩略图

我也是 请问你解决了吗

**Reply by sangchengfang** [reply]
> > 我也是，打开开关也不能生成缩略图
> 
> 我也是 请问你解决了吗

用ffmpeg版本的docker镜像
